pkg/webhook/api_debug_server.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package webhook
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"k8s.io/klog/v2"
+)
+
+var debugApiProviderMap = map[string]DebugApiProvider{}
+
+type DebugApiProvider interface {
+	RegisterDebugApi(engine *gin.Engine)
```

Comment:
Consider not passing `gin.Engine` as a parameter directly, instead passing `gin.RouterGroup`

Reply:
fixed